react/src/pages/EnvironmentPage.tsx

```diff
@@ -45,38 +62,40 @@ const EnvironmentPage = () => {
         },
       }}
     >
+      {curTabKey === 'image' ? (
+        <Flex
+          style={{
+            display: 'block',
+            paddingTop: token.paddingContentVerticalSM,
```

Comment:
```suggestion
```

This code seems to break the consistency of the css styles. It would be better to control it inside the `ImageList`.